openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -1165,17 +1165,12 @@
 - control_key: AC-10
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: unsatisfied
```

Comment:
@carlosmmatos is this status the correct one in case a product doesn't have the capability to address a control at the moment?

Reply:
Thought that "planned" was appropriate, since there is an RFE there to point to. And it'll get addressed...at some point.

Reply:
I'm fine changing to planned, I wasn't sure if planned just pertains to the reply (so..it would say, we plan to deliver a reply) or also the result of the reply (we plan to deliver a change in openshift that satisfies the control)

Reply:
@jhrozek Planned is appropriate if we intend to address this at some point during the foreseeable future. If we don't plan on ever doing this, then we would need to provide verbiage as to why we don't meet this, and don't think that we need to implement it.

Reply:
thanks, reverted to planned (I really hope this gets addressed)